reference/artifacts/types/s3-object/index.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: single
+title:  "S3 Object"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+[S3](https://aws.amazon.com/s3/) is an object store, and S3 object
+artifacts are references to objects stored in S3 buckets. These artifacts are
+generally consumed by stages that read configuration from text files, such as a
+Deploy Manifest or AWS Deploy stage.
+
+## Fields
+
+| Field | Explanation |
+|-|-----------|
+| `type` | Always `s3/object`. |
+| `name` | Name/Path of the object. |
+| `location` | The region where your bucket was created |
+
+## Example
+
+```js
+{
+  "type": "s3/object",
+  "name": "s3://bucket/file.json",
+  "location": "us-east-1"
```

Comment:
In the example you added to /reference/artifacts/index.md you included `reference` and `version` fields. Should those be added here, and to the list of fields above?

Reply:
So it's actually the other way around. S3 doesn't version the files the same way, so those aren't used here. Updated in the example